### DISCUSSION TITLE
items for 0 euros

### DISCUSSION THREAD

**Discussion by Oleafeon** [duncanmcclean/simple-commerce#1190] [Help] [answered]
In my webshop i have some items which are 0,00 euros but when i try to click add to cart those items are not added

**Comment by duncanmcclean** [comment]
What does your "add to cart" form look like?

How are you checking if the line items are being added to the cart? Viewing the cart/order in the CP?

**Comment by Oleafeon** [comment]
Hey @duncanmcclean 

here you go, and i am checking inside my bag but the 0 euros items are not added

```
{{ sc:cart:addItem }}
<input type="hidden" name="product" value="{{ id }}">
<input type="hidden" name="quantity" value="1">
<button class="btn btn-custom">
    <i class="fas fa-shopping-cart"></i> {{ add_to_bag }}
</button>
{{ /sc:cart:addItem }}
```

**Reply by duncanmcclean** [reply]
Does the product you're trying to add have any variants?

**Reply by Oleafeon** [reply]
nope

**Reply by duncanmcclean** [reply]
If you add `{{ sc:errors | dump }}` to your template, and submit the form, do you get any errors in the dump?

**Reply by Oleafeon** [reply, marked as answer]
Okay weirdly enough but now the 0.00 is working it add the free product to my cart
